README.md

```diff
@@ -1,4 +1,8 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
We can leave this line for now. Credit where credit's due. Just move it below the setup instructions.